public/app/plugins/datasource/loki/querybuilder/components/NestedQuery.tsx

```diff
@@ -70,7 +71,7 @@ export const NestedQuery = React.memo<Props>(
             />
           </div>
           <FlexItem grow={1} />
-          <IconButton name="times" size="sm" onClick={() => onRemove(index)} />
+          <IconButton ariaLabel="remove-nested-query" name="times" size="sm" onClick={() => onRemove(index)} />
```

Comment:
Good idea adding an `aria-label`. Always keep in mind that `aria-` attributes are meant for accessibility assistants like screen readers. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

So:
```suggestion
          <IconButton ariaLabel="Remove nested query" name="times" size="sm" onClick={() => onRemove(index)} />
```